### DISCUSSION TITLE
Not admin

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [victornpb/undiscord#609] [1. Questions & Answers]
Can I also use this to delete only my messages while not being server admin?

**Comment by viiper2000** [comment]
I'm pretty sure you can, unless the server owner has manage messages perms turned off. Just mind you that you'll ONLY be deleting your messages, and that's the way you should be using this anyway.

Deleting everyone's messages, I would imagine, is a pretty easy way to get detected. Don't set the delete delay below like 2000ms either.
